src/components/chat/chat-log.tsx

```diff
@@ -3,7 +3,7 @@ import cx from 'classnames'
 
 export type ChatLogEntry = {
```

Comment:
Hier wär's vielleicht in Zukunft cool den Union Type besser aufzuziehen:

```suggestion
export type ChatLogEntry = UserJoined | UserLeft | UserReacted | TrackSkipped

type UserReacted = {
  id: string
  timestamp: number
  type: 'USER_REACTED',
  reaction: 'LOVE' | 'HATE' | 'FLAMES' | 'CELEBRATE'
}
```

Dann könnte man beim rendern der Message auf den Type und dann auf die Art der Reaction switchen und die Event logik würde dann nicht komplizierter werden (Vielleicht? Vielleicht auch zu kompliziert gedacht.. 😅 )